src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Include name="beta-note.mdx" />
+
+You can monitor the performance of your views in a SwiftUI project with SentrySwiftUI library.
+
+Install it with CocoaPod:
```

Comment:
If you apply this suggestion you can ditch `Or by importing sentry-cocoa repository (https://github.com/getsentry/sentry-cocoa) with SPM.` below
```suggestion
You can monitor the performance of your views in a SwiftUI project with the SentrySwiftUI library, which you can install via <PlatformLink to="/install/swift-package-manager/">SPM (Swift package manager)</PlatformLink> or CocoaPods:
```